lib/mocks/mock_location.dart

```diff
@@ -11,14 +11,111 @@ class MockLocation extends Location {
           LocationFact(
               title: 'How to get there', text: 'Take a taxi from Busia')
         ]),
+    Location(
+        name: ' Kabale town, Uganda',
```

Comment:
Remove the word town, and the country because they are redundant as appropriate in the all locations.